kernel/src/process.rs

```diff
@@ -57,16 +72,22 @@ pub struct Callback {
 }
 
 #[repr(C)]
+#[derive(Clone, Copy, Debug)]
 struct LoadInfo {
-    total_size: usize, // Total padded size of the program image
-    rel_data_size: usize,
-    entry_loc: usize, // Entry point for user application
-    init_data_loc: usize, // Data initialization information in flash
-    init_data_size: usize, // Size of initialization information
-    got_start_offset: usize, // Offset to start of GOT
-    got_end_offset: usize, // Offset to end of GOT
-    bss_start_offset: usize, // Offset to start of BSS
-    bss_end_offset: usize, // Offset to end of BSS
+    total_size: u32,
+    entry_offset: u32,
+    rel_data_offset: u32,
+    rel_data_size: u32,
+    text_offset: u32,
+    text_size: u32,
+    got_offset: u32,
+    got_size: u32,
+    data_offset: u32,
+    data_size: u32,
+    bss_start_offset: u32,
```

Comment:
How about `bss_mem_offset`?

Everything else refers to offset in the file, while this field refers to offset in the process memory.